genome_kit/gk_data.py

```diff
@@ -56,6 +56,17 @@ def get_file(filename):
 def upload_file(filepath:str, filename:str, metadata:Dict[str, str]=None):  # pragma: no cover
     """compatibility wrapper for :py:meth:`~genome_kit.DataManager.upload_file`"""
 
+    chrom_sizes_ext = ".chrom.sizes"
+    if filename.endswith(chrom_sizes_ext):
+        refg_name = filename[:-len(chrom_sizes_ext)]
+        hashval = _cxx.Genome._refg_hash(refg_name)
```

Comment:
Can probably just do Genome._refg_hash but either is fine

Reply:
If I `from . import Genome` I get a circular dependency.